nncf/common/graph/utils.py

```diff
@@ -89,3 +91,57 @@ def get_split_axis(input_shapes: List[List[int]], output_shapes: List[List[int]]
         logger.warning('Identity split/concat node detected')
 
     return axis
+
+
+def check_scope_names_match_graph(config: NNCFConfig, graph: NNCFGraph) -> None:
```

Comment:
>Should we expose this on the API level? Or the idea is that the user should play with the config to find the right scope?
Yes, the idea is that user should run NNCF only with correct config, otherwise we stop process as soon as possible.

Reply:
@AlexKoff88, @AlexanderDokuchaev I don't see how it can be used in the PTQ API.